customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDao.java

```diff
@@ -67,6 +68,7 @@ public class CustomerDao implements Typed {
     private DoiAgentDao doiAgent;
     private boolean nviInstitution;
     private boolean rboInstitution;
+    private Boolean allowsGeneralSupport;
```

Comment:
A boolean value, when we probably want something else, but OK.

Reply:
This is made to made to be able to set this field to `true` for all all current customers. Otherwise it will be set to `false` by default. I can "map" it change it boolean where everything is migrated